packages/mobx/__tests__/v5/base/typescript-tests.ts

```diff
@@ -204,32 +208,6 @@ test("scope", () => {
     t.equal(x3.z, 8)
 })
 
-test("typing", () => {
-    const ar: IObservableArray<number> = observable([1, 2])
-    mobx.intercept(ar, (c: IArrayWillChange<number> | IArrayWillSplice<number>) => {
-        // console.log(c.type)
-        return null
-    })
-    mobx.observe(ar, (d: IArrayDidChange<number>) => {
-        // console.log(d.type)
-    })
-
-    const ar2: IObservableArray<number> = observable([1, 2])
-    mobx.intercept(ar2, (c: IArrayWillChange<number> | IArrayWillSplice<number>) => {
-        // console.log(c.type)
-        return null
-    })
-    mobx.observe(ar2, (d: IArrayDidChange<number>) => {
-        // console.log(d.type)
-    })
-
-    const x: IObservableValue<number> = observable.box(3)
-})
-
-const state: any = observable({
```

Comment:
@urugator This variable isn't used anywhere. It also doesn't do anything useful because `observable` gets casted to `any`